contents/blog/posthog-first-five.md

```diff
@@ -1,6 +1,6 @@
 ---
 date: 2022-08-16
-title: What we learned about hiring from our "first five" employees
+title: What we learned about hiring from our first five employees
```

Comment:
You should be able to encapsulate the entire thing, then use quotes inside if it helps!

Reply:
Oh no, we removed for stylistic reasons rather than techincal.